dependencies/socketw/sw_base.h

```diff
@@ -18,7 +18,9 @@
 
 #include "sw_internal.h"
 
-#include <unistd.h>
+#if !(_MSC_VER == 1900) // Visual Studio 2015 has no <unistd.h>
+#   include <unistd.h>
```

Comment:
It should probably include all MS VS versions since unistd.h is only available on POSIX compliant systems as well as MinGW and Cygwin. [1]

[1] https://en.wikipedia.org/wiki/Unistd.h